awspub/configmodels.py

```diff
@@ -94,6 +95,34 @@ class ConfigImageSSMParameterModel(BaseModel):
     )
 
 
+class ConfigSNSNotificationProtocol(str, Enum):
```

Comment:
i would drop `Config`. its not really a config like object, just an enum to advertise what time of protocols are supported

Reply:
Dropped `Config` in the class name.